xml/admin_saltcluster.xml

```diff
@@ -1141,67 +1142,87 @@ OSD_ID  HOST         STATE                    PG_COUNT  REPLACE  FORCE  STARTED_
    </step>
    <step>
     <para>
-     If the &adm; is a &smaster; at the same time, stop
-     <systemitem class="daemon">salt-master</systemitem>
-     <systemitem class="daemon">salt-minion</systemitem> services on the old
-     &adm;:
+     If the old &smaster; is also the only administration node in the cluster,
+     then manually move <filename>/etc/ceph/ceph.client.admin.keyring</filename>
+     and <filename>/etc/ceph/ceph.conf</filename> to the new &smaster;.
     </para>
+    </step>
+    <step>
+      <para>
+        Stop and disable the &smaster; &systemd; service on the old &smaster; node:
+      </para>
 <screen>
 &prompt.smaster;systemctl stop salt-master.service
+&prompt.smaster;systemctl disable salt-master.service
+</screen>
+   </step>
+   <step>
+     <para>
+       If the old &smaster; node is no longer in the cluster, also stop and
+       disable the &sminion; &systemd; service:
+     </para>
+<screen>
 &prompt.smaster;systemctl stop salt-minion.service
+&prompt.smaster;systemctl disable salt-minion.service
 </screen>
+    <warning>
+      <para>
+        Do not stop or disable the <literal>salt-minion.service</literal> if
+        the old &smaster; node has any &ceph; daemons (MON, MGR, OSD, MDS,
+        gateway, monitoring) running on it.
+      </para>
+    </warning>
    </step>
    <step>
     <para>
-     Install &cephos; on the new &adm; following the procedure described in
+     Install &cephos; on the new &smaster; following the procedure described in
      <xref linkend="deploy-os"/>.
     </para>
    </step>
    <step>
-    <para>
-     If the &adm; is a &smaster; at the same time, install the
-     <package>salt-master</package> (and possibly
-     <package>salt-minion</package>) package. If the &adm; is
-     <emphasis>not</emphasis> a &smaster;, install the
-     <package>salt-minion</package> package only. Configure &salt; so that the
-     &smaster; and &sminion;s communicate. Follow the procedure described in
-     <xref linkend="deploy-salt"/>.
-    </para>
-    <tip>
-     <title>Transition of Salt Minions</title>
```

Comment:
We are moving the salt Master node to a different node, which will require that the new &smaster; and &sminion;s can communicate. So, is there any reason to remove this information from docs?

Reply:
It's currently situation as a "tip". This can be moved back into line 1178 as a tip again, or as a separate instruction. What do you advise?

Reply:
A tip sounds good.